pr_watch/migrations/0012_auto_20210104_1044.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.12 on 2021-01-04 10:44
```

Comment:
@gabor-boros I think there is already a 0012 migration, please rebase on top of that one. Also could you please set a name to this migration as well? So it can be more understandable (`--name`)

Reply:
👌 will rename tomorrow morning

Reply:
Renamed 😊